### PR TITLE
Meeting top bar avatars mk

### PR DESCRIPTION
**Pull request by mattkrick**
@ackernaut little fixups for the PR.
I think a nice addition would be to add animations when people connect/disconnect. we can copy the code from the atlassian domain icons. the overflow number could roll up when incremented & down when decremented.

### CODE REVIEW

File: packages/client/components/Facilitator.tsx [hunk 1]
```diff
@@ -141,9 +142,11 @@ export default createFragmentContainer(Facilitator, {
         newMeeting {
           facilitatorUserId
           facilitator {
-            isConnected
             picture
             preferredName
+            user {
```
Comment (ackernaut) [inline packages/client/components/Facilitator.tsx]: @mattkrick what’s the difference between these 2?

Comment (ackernaut) [reply packages/client/components/Facilitator.tsx]: ah looks like a deprecated pattern that you are removing, thanks!!

File: packages/client/modules/meeting/components/MeetingAvatarGroup/NewMeetingAvatarGroup.tsx [hunk 2]
```diff
@@ -75,62 +76,45 @@ interface Props {
...
+  const visibleConnectedTeamMembers = connectedTeamMembers.slice(0, overflowThreshold)
+  const hiddenTeamMemberCount = connectedTeamMembers.length - visibleConnectedTeamMembers.length
   return (
     <MeetingAvatarGroupRoot>
       <VideoControls
         allowVideo={allowVideo}
         swarm={swarm}
         localStreamUI={camStreams[atmosphere.viewerId]}
       />
-      {self && (
```
Comment (ackernaut) [inline packages/client/modules/meeting/components/MeetingAvatarGroup/NewMeetingAvatarGroup.tsx]: @mattkrick I like what you’re doing here. One thing, the intent was to always at least show the local user’s avatar, not just the 1st of the connected team members via check-in sort order.

Comment (ackernaut) [reply packages/client/modules/meeting/components/MeetingAvatarGroup/NewMeetingAvatarGroup.tsx]: I guess you’re checking that with `a.userId === viewerId ?` on 92? Testing locally now…

**Comment by ackernaut** [comment]
@mattkrick this looks good, should we add animations here or on the main branch?

File: packages/client/modules/meeting/components/MeetingAvatarGroup/NewMeetingAvatarGroup.tsx (hunk 2 above)
Comment (mattkrick) [reply packages/client/modules/meeting/components/MeetingAvatarGroup/NewMeetingAvatarGroup.tsx]: yep! the tricky part is that the viewer may not be connected & we don't want to show them as connected if they aren't.

i guess we should design out what it should look like when the viewer isn't connected? maybe...
- don't show their avatar
- show others' avatars as faded
- show the overflow count avatar as gray

**Comment by mattkrick** [comment]
i think we should merge now, but have animations be top priority since the list now changes & we need to draw attention to it a lot more.

**Comment by ackernaut** [comment]
Good deal, I write up an enhancement issue.

**Pull request merged**